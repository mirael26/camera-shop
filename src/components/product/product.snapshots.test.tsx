import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import { productMock } from '../../test/mocks';
import Product from './product';

jest.mock('../rating-stars/rating-stars', () => 'RatingStars');

test('Product renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Product/>, { initialState: { 
    data: { currentProduct: productMock }
  }});
  expect(asFragment()).toMatchSnapshot();
});
