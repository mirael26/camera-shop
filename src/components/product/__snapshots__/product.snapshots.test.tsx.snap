// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Render correctly 1`] = `
<DocumentFragment>
  <div
    class="page-content__section"
  />
</DocumentFragment>
`;

exports[`Product renders correctly 1`] = `
<DocumentFragment>
  <div
    class="page-content__section"
  />
</DocumentFragment>
`;
